### ISSUE TITLE
Native Distribution

### ISSUE THREAD

**Issue by mehmetakify** [realm/realm-kotlin-samples#33]
After creating native distribution, installed apps not working. I tried Mac and Linux desktops with dmg and deb files. Adding working examples to samples would be great.

**Comment by cmelchior** [comment]
When you say "creating a native distribution" what do you mean exactly?

**Comment by mehmetakify** [comment]
I mean creating .exe, .dmg, .deb files to publish and share desktop app.

You can see more detailed explanation [compose desktop](https://github.com/JetBrains/compose-jb/blob/master/tutorials/Native_distributions_and_local_execution/README.md)

I changed build.gradle.kts following instructions from the link and successfully create .dmg file, but after install the application app does not work.

Main reason is realm causes some read-write permission error at mac devices and I need working example.

In short, how can I create working .dmg files of realm-kotlin-samples?
